lib/menu.rb

```diff
@@ -0,0 +1,75 @@
+require_relative 'order'
+
+class Menu
+
+  attr_reader :menu
+
+  def initialize
+    @menu = [{ :name => "Risotto", :price => 7.99 },
+      { :name => "Large Pizza", :price => 12.99 },
+      { :name => "Fried Chicken", :price => 6.99 },
+      { :name => "Green Salad", :price => 5.99 },
+      { :name => "Pasta", :price => 8.99 },
+      { :name => "Vegetable Curry", :price => 6.99 },
+      { :name => "Rice", :price => 1.99 },
+      { :name => "Steamed Vegetables", :price => 1.99 },
+      { :name => "Acai Bowl", :price => 3.99 }]
```

Comment:
Very nice and tidy. I prefer to put long arrays in this format:
```menu = [
{ name:  'Risotto', price: 7.99 }
ect...
]```

Reply:
Yeah definitely something for my refactoring pass